flowmachine/flowmachine/features/location/redacted_in_out_flows.py

```diff
@@ -0,0 +1,23 @@
+from flowmachine.core import Query
+from flowmachine.features.location.flows import BaseInOutFlow
+from flowmachine.features.location.redacted_location_metric import (
+    RedactedLocationMetric,
+)
+
+
+class RedactedInOutFlow(RedactedLocationMetric, BaseInOutFlow, Query):
```

Comment:
Note you'll also lose the index columns (although one could fix that in `RedactedLocationMetric` as well

Reply:
Just catching up on this conversation. It looks to me like there's currently no need for `RedactedInOutFlow` to inherit from `BaseInOutFlow`, but fixing https://github.com/Flowminder/FlowKit/issues/5132 will require overriding `_geo_augmented_query` in `BaseInOutFlow`, and that will also be required here. So the current inheritance makes sense in the long run.

Reply:
Hmm. I disagree - the in/outflows shouldn't be retaining the from/to suffix to the location columns, rather than needing an override of the geojsoning.

Reply:
Ah, true. Without the from/to suffix the in/outflows are just like the result of any other spatial aggregate, so then there's no special treatment required and redacted in/outflows should just need the `RedactedLocationMetric` and `GeoDataMixin` mixins, the same as other metrics.

There's the issue that `BaseInOutFlow` also handles labelled in/outflows, and the default geojson representation isn't ideal for that (each geography will be duplicated `n_labels` times in the output).  But that's an issue more widely than just for in/outflows (e.g. `LabelledSpatialAggregate` has the same issue), and it isn't relevant for this PR.

Reply:
Yup, https://github.com/Flowminder/FlowKit/commit/c5a780163e92cad7152d651181f11c3b258fbe0f is my proposed solution to that.